components/ui/NavBar.tsx

```diff
@@ -1,9 +1,42 @@
+'use client';
+
 import Link from 'next/link'
 
 import { siteConfig } from 'config/site'
 import { NavItem } from 'types'
+import { useEffect, useState } from 'react'
+import { LoginModal } from 'components/chapters/LoginModal';
+import Image from 'next/image';
+import User from 'public/assets/icons/user.svg';
```

Comment:
We already have avatar.svg in the repo, you can use that.

Reply:
Done